docs/topics/feed-exports.rst

```diff
@@ -389,6 +523,20 @@ as a fallback value if that key is not provided for a specific feed definition:
 
     .. versionadded:: 2.3.0
 
+-   ``batch_file_size``: falls back to
+    :setting:`FEED_EXPORT_BATCH_FILE_SIZE`.
+
+    .. versionadded:: VERSION
+
+-   ``batch_duration``: falls back to
+    :setting:`FEED_EXPORT_BATCH_DURATION`.
+
+    .. versionadded:: VERSION
+
+-   ``batch``: a custom batch handler to handle batch deliveries.
+
+    .. versionadded:: VERSION
+
```

Comment:
In the documentation about the batch settings, it should be clear that they only apply if using the default batch handler.